pkg/utils/kubernetes/controllers.go

```diff
@@ -87,7 +93,7 @@ var APIGroupControllerMap = map[string]map[string]versionutils.VersionRange{
 		"endpointslice":                        {},
 		"endpointslicemirroring":               {},
 		"ephemeral-volume":                     {},
-		"garbagecollector":                     {},
+		"garbagecollector":                     {RemovedInVersion: "1.30"},
```

Comment:
This looks strange 👀 Which kind of garbage collector was removed here? Probably not the general one responsible for `ownerReferences`, etc.? 😕

Reply:
`garbagecollector` is not removed, the controller just doesn't vendor `core/v1` after https://github.com/kubernetes/kubernetes/pull/123344, so we don't disable the controller when `core/v1` API group is disabled in the Shoot spec.